app/models/bishop.rb

```diff
@@ -1,7 +1,7 @@
 class Bishop < ChessPiece
-  def valid_move?(destination_x, destination_y)
-    return false if out_of_bounds?(destination_x, destination_y)
-    return false unless diagonal_move?(destination_x, destination_y)
-    !(obstructed?(destination_x, destination_y))
+  def valid_move?(coords)
```

Comment:
I'd spell it out `coordinates`

Someone new coming into the codebase might go "what the heck is a coord"?